content/influxdb/v2.0/reference/glossary.md

```diff
@@ -903,36 +909,45 @@ Service input plugins listen on a socket for known protocol inputs, or apply the
 
 Related entries: [aggregator plugin](#aggregator-plugin), [input plugin](#input-plugin), [output plugin](#output-plugin), [processor plugin](#processor-plugin)
 
-<!--### shard
+### shard
 
-A shard contains encoded and compressed data. Shards are represented by a TSM file on disk.
-Every shard belongs to one and only one shard group.
-Multiple shards may exist in a single shard group.
-Each shard contains a specific set of series.
-All points falling on a given series in a given shard group will be stored in the same shard (TSM file) on disk.
+A shard contains encoded and compressed data for a specific set of [series](#series).
+A shard consists of one or more [TSM files](#tsm-time-structured-merge-tree) on disk.
+All points in a series in a given shard group are stored in the same shard (TSM file) on disk.
+A shard belongs to a single [shard group](#shard-group).
 
-Related entries: [series](#series), [shard duration](#shard-duration), [shard group](#shard-group), [tsm](#tsm-time-structured-merge-tree)
+For more information, see [Shards and shard groups (OSS)](/influxdb/%762.0/reference/internals/shards/).
 
-### shard duration
+Related entries: [series](#series), [shard duration](#shard-duration),
+[shard group](#shard-group), [tsm](#tsm-time-structured-merge-tree)
 
-The shard duration determines how much time each shard group spans.
-The specific interval is determined by the `SHARD DURATION` of the retention policy.
-<!-- See [Retention Policy management](/{{< latest "influxdb" "v1" >}}/query_language/manage-database/#retention-policy-management) for more information.
+### shard group
 
-For example, given a retention policy with `SHARD DURATION` set to `1w`, each shard group will span a single week and contain all points with timestamps in that week.
+Shard groups are logical containers for shards organized by [bucket](#bucket).
+Every bucket that contains data has at least one shard group.
+A shard group contains all shards with data for the time interval covered by the shard group.
+The interval spanned by each shard group is the [shard group duration](#shard-group-duration).
```

Comment:
Does it make sense/streamline to combine lines 928-929 with line 926? Something like:

_Shard groups are organized by [bucket](#bucket) and contain all shards with data for a specified interval known as the [shard group duration](#shard-group-duration)._